### DISCUSSION TITLE
Is it possible to sync *all* Taskwarrior tasks?

### DISCUSSION THREAD

**Discussion by shymega** [bergercookie/syncall#102] [Q&A]
HI!

Great project (albeit I have to use it from Git, due to the Cython3 PyYAML dep) so far.

I'm having trouble syncing all of my Taskwarrior tasks - I wanted to select everything to sync to Google Tasks.

Is that possible?

Thanks :)

**Comment by bergercookie** [comment]
Hi @shymega , thanks for the kind words!

Currently not possible without code changes, but I'm planning to implement this in the coming weeks so stay tuned.

**Reply by shymega** [reply]
No worries. I understand the motivation kind words can have on a project. At least from maintaining several projects myself.

Thank you!

**Comment by bergercookie** [comment]
See ticket for more #103 for more info on this
